### PR TITLE
POL-1338 Validated Permissions

### PR DESCRIPTION
**Pull request by XOmniverse**
### Description

This PR makes several changes related to tracking policy permissions:

- Several policy templates that were missing have been validated and added. Where appropriate, these policy templates and their associated README files were updated.
- Automation has been added to track every non-deprecated policy template that is not in the validation list. This is to assist in completing this project by getting all of the missing policy templates added.
- A couple of deprecated policy templates were missing the deprecated: "true" field in the info block. This has been fixed.
- Minor tweaks made to changed files to pass current Dangerfile tests

### CODE REVIEW

File: .spellignore [hunk 1]
```diff
@@ -565,3 +565,17 @@ USD
...
+licenseId
+licenseName
+licenseStatus
+licenseType
+Visio
+Orgs
+Org
+RightScale
+MCA
+IOT
```
Comment (nia-vf1) [inline .spellignore:579]: Should be IoT rather than IOT I would have thought

File: cost/flexera/cco/scheduled_report_markupsdowns/README.md [hunk 2]
```diff
@@ -45,11 +38,20 @@ Leave the field blank to report on all top level Billing Centers.
 - *IOT markup or markdown percentage* - markup for the IOT category in number value (20 being 20% markup -20 being a 20% markdown. Leave 0 to apply the general markup).
```
Comment (nia-vf1) [inline cost/flexera/cco/scheduled_report_markupsdowns/README.md:38]: Same as IoT comment in .spellignore file - I think this should be IoT rather than IOT

Comment (XOmniverse) [reply cost/flexera/cco/scheduled_report_markupsdowns/README.md:38]: This appears to be based on a category that appears in our platform as IOT

File: .spellignore (hunk 1 above)
Comment (XOmniverse) [reply .spellignore:579]: This appears to be based on a category that appears in our platform as IOT

**Review by nia-vf1 — APPROVED** [review]
LGTM

**Pull request merged**